libs/@guardian/source-development-kitchen/src/react-components/ticker/Ticker.tsx

```diff
@@ -0,0 +1,167 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { textSans15, textSansBold17 } from '@guardian/source/foundations';
+import { useEffect, useState } from 'react';
+import { useIsInView } from './useIsInView';
+import { useTicker } from './useTicker';
+
+interface TickerCopy {
+	countLabel: string;
+}
+
+export interface TickerData {
+	total: number;
+	goal: number;
+}
+
+export interface TickerStylingSettings {
+	headlineColour: string;
+	totalColour: string;
+	goalColour: string;
+	filledProgressColour: string;
+	progressBarBackgroundColour: string;
+}
+
+export interface TickerSettings {
+	currencySymbol: string;
+	copy: TickerCopy;
+	tickerData: TickerData;
+	tickerStylingSettings: TickerStylingSettings;
+}
+
+//styles for the container that holds the ticker
+const tickerContainerStyles = css`
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	padding: 20px 8px 20px 8px;
```

Comment:
Where possible I think it's preferable to use `space` from `@guardian/source/foundations` so that our spacing conforms to the design system and can be updated in one place.

https://github.com/guardian/source/blob/d84c25d67ed51545994f53986eb4487d35c1aab4/packages/%40guardian/source-foundations/src/space/space.ts#L19

Example usage in DCR:

https://github.com/guardian/dotcom-rendering/blob/9212caca630ce238d67307799facf07afe108abf/dotcom-rendering/src/components/Accordion.tsx#L49

Reply:
Yeah really good point. I've used `space` where I can although some of the spacing on the design is custom I can't be used like 7px